Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -305,6 +305,21 @@ namespace AzToolsFramework
             return GetEntryFromIndex<SourceAssetBrowserEntry>(index) == nullptr;
         }
 
+        void AssetBrowserTreeView::OpenItemForEditing(const QModelIndex& index)
+        {
+            QModelIndex proxyIndex = m_assetBrowserSortFilterProxyModel->mapFromSource(index);
+
+            if (proxyIndex.isValid())
+            {
+                selectionModel()->select(proxyIndex, QItemSelectionModel::ClearAndSelect);
```

Comment:
is it possible to select/open multiple assets at once in the AB? Great if not, but if it is, this will only select the most recently opened link

Reply:
It is not currently possible and I haven't head of any plans for it to be.